website/docs/getting_started/getting_started.md

```diff
@@ -0,0 +1,159 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+slug: /getting_started
+---
+
+## Purchase a Radio
+
+The easiest way is to [buy a device with the software already installed](https://www.aliexpress.com/item/4001178678568.html) In the Americas get the 915mhz version, In Europe the 868Mhz or Asia 923Mhz see [here](https://www.thethingsnetwork.org/docs/lorawan/frequencies-by-country.html).
```

Comment:
Maybe "While there are a number of different [hardware devices] (link to hardware overview) you can use, the easiest way is to...."?

"For other countries see thethingsnetwork list of [available frequencies] (link)"

Reply:
I'm kind of waiting to build a purchase page to have things really clearly stated on this one. I'd like to have a list of all the radios using Meshtastic affiliate links so a portion of the purchase comes back to the project. So I didn't really do too much with this besides copy paste. Also, do we want to have the thingsnetwork link? Or write something internally?

I can add a 
```
<!--- TODO clean this section up and use affiliate links --->
```

Reply:
Waiting for the purchase page makes sense.

 I think after mentioning the main three frequency areas, thethingsnetwork link is probably the easiest way of ensuring that anyone can find their specific frequency.